src/runtime/providers/directus.ts

```diff
@@ -8,8 +8,10 @@ export const operationsGenerator = createOperationsGenerator({
 
 export const getImage: ProviderGetImage = (src, { modifiers = {}, baseURL } = {}) => {
   // Separating the transforms from the rest of the modifiers
-  const transforms = modifiers.transforms
+  let transforms = modifiers.transforms
   if (transforms && transforms.length > 0) {
+    // de-duplicate (can get multiplied when haveing >1 densities configured)
+    transforms = transforms.filter((value: any, index: any) => transforms.indexOf(value) === index)
```

Comment:
I think we might need to try JSON.stringify on the values to be compared as I think we're not always guaranteed they are the _same_ array.